app/javascript/src/utils/pusher.js

```diff
@@ -1,35 +1,35 @@
 // import Pusher from 'pusher-js';
 
 export default function setupPusher(key, callbackObjects) {
-  let pusher;
-  return import('pusher-js')
-    .then(({ }) => {
-    if (window.pusher) {
-      pusher = window.pusher
-    } else {
+  return import('pusher-js').then(_ => {
+    const {
```

Comment:
Leveraging [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here and default values for the destructured variables.